test/algorithms/similarity/discrete_frechet_distance.cpp

```diff
@@ -0,0 +1,105 @@
+// Boost.Geometry
+
+// Copyright (c) 2018 Yaghyavardhan Singh Khangarot, Hyderabad, India.
+
+// Contributed and/or modified by Yaghyavardhan Singh Khangarot, as part of Google Summer of Code 2018 program.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#include <vector>
+
+#define BOOST_GEOMETRY_TEST_DEBUG
```

Comment:
This should not be defined in the cpp file. This is not a part of the standard testing. You should enable it in the compiler if needed.